models/baseModels/Invoice/Invoice.ts

```diff
@@ -553,6 +588,17 @@ export abstract class Invoice extends Transactional {
       },
       dependsOn: ['party'],
     },
+    loyaltyProgram: {
+      formula: async () => {
+        const LoyaltyPointsData = await this.fyo.doc.getDoc(
+          ModelNameEnum.Party,
+          this.party
+        );
+        const loyaltyProgram = LoyaltyPointsData?.loyaltyProgram as string;
+        return loyaltyProgram ?? '';
+      },
```

Comment:
if it is already asserted as `string`, no need to return `loyaltyProgram ?? ''` .. simply just return `loyaltyProgram` only.